src/CoolWifi.cpp

```diff
@@ -628,3 +628,47 @@ bool CoolWifi::addWifi(String ssid, String pass) {
 
   return (true);
 }
+
+
+/**
+ *  CoolWifi::getExternalIP():
+ *  This method is provided to print the
+ *  public IP of a device to a String
+ */
+String CoolWifi::getExternalIP()
+{
+#if DEBUG == 1
+
+  Serial.println("Entering CoolWifi.addWifi() ");
+
+#endif
+
+  WiFiClient client;
+  String IP;
+  if (!client.connect("api.ipify.org", 80)) {
+    Serial.println(F ("Failed to connect with 'api.ipify.org' !"));
+  }
```

Comment:
per LLVM specifications (and almost all others), braces for `else` statement should look like 
```
} else {
```
not
```
}
else {
```